ush/land/letkf_create_ens.py

```diff
@@ -0,0 +1,36 @@
+import os
+import numpy as np
+from netCDF4 import Dataset
+import sys
+# Can silence the DeprecationWarning by add the following:
+# from warnings import filterwarnings
+# filterwarnings(action='ignore', category=DeprecationWarning, message='`np.bool` is a deprecated alias')
+# routine to create pseudo-ensemble for use in LETKF-OI for snow depth
+# reads in sfc_data restarts, and sets snowdepth to +/-
+# perturbation, given stdev(ensemble) of B
+# Clara Draper, October, 2021.
+
+if (len(sys.argv) != 4):
+    print('argument error, usage: letkf_create file_stub back_error')
+
+fstub = sys.argv[1]
+b = float(sys.argv[2])
+workdir = sys.argv[3]
+
+# 2 ens members
+offset = b/np.sqrt(2)
+
+print(f"adjusting {fstub}* by {str(offset)}")
```

Comment:
do you need the `str()` here? Or is this for having too many significant figures?

Reply:
No, and I removed `str()`.